pyproject.toml

```diff
@@ -153,6 +154,37 @@ allow_untyped_defs = true
 allow_incomplete_defs = true
 allow_untyped_calls = true
 
+[tool.ruff]
+line-length = 100
+lint.select = [
+    "ASYNC",
+    "B",
+    "C4",
+    "C90",
+    "E",
+    "F",
+    "FLY",
+    "PIE",
+    "S",
+    "W",
+]
+lint.ignore = [
+    "B017",
+    "B028",
+    "B904",
+    "E203",
+    "E501",
+]
+lint.exclude = [ "_vendored" ]
+lint.mccabe.max-complexity = 91  # Default is 10
+
+[tool.ruff.lint.per-file-ignores]
+"isort/hooks.py" = [ "S603" ]
+"isort/settings.py" = [ "S603", "S607" ]
+"tests/*" = [ "S" ]
+"tests/unit/example_crlf_file.py" = [ "F401" ]
+"tests/unit/test_wrap_modes.py" = [ "PIE804" ]
```

Comment:
Similar question as above, why are we ignoring these here?

Reply:
Similar position.  Let's land the tool first before we tighten up the config.